typescript/infra/src/config/gas-oracle.ts

```diff
@@ -230,6 +231,10 @@ export function getTokenExchangeRateFromValues(
   remote: ChainName,
   tokenPrices: ChainMap<string>,
 ): BigNumber {
+  if (!tokenPrices[local] || !tokenPrices[remote]) {
```

Comment:
some dependency issue here

goal is to provide some default value to satisfy the config on load, knowing it will get overwritten when the script actually gets run

Reply:
can you put the reason into a comment so this doesn't get forgotten